app/models/manageiq/providers/ibm_power_hmc/inventory/collector/infra_manager.rb

```diff
@@ -11,27 +11,27 @@ def collect!
   def hmc
     @hmc ||= begin
       connection.management_console
-    rescue IbmPowerHmc::Connection::HttpError => e
+    rescue => e
       $ibm_power_hmc_log.error("management console query failed: #{e}")
```

Comment:
Completely unrelated but if all we're doing here is logging on errors I wonder if we could add a Logger to the sdk gem and set it to `$ibm_power_hmc`

Reply:
Good idea. Maybe we can open a separate issue to do it as part of future enhancements.
These errors are also logged by EVM but the HMC provider logs have less traffic, so it's easier to spot 😊.
They also appear in UI, but they are often truncated:

![image](https://user-images.githubusercontent.com/48122102/198646224-4a02983d-bf75-4081-bdce-7ea5e2444a74.png)